compiler/rustc_session/src/config/cfg.rs

```diff
@@ -413,7 +413,7 @@ impl CheckCfg {
                 ] = self
                     .expecteds
                     .get_many_mut(VALUES)
-                    .expect("unable to get all the check-cfg values buckets");
+                    .map(|x| x.expect("unable to get all the check-cfg values buckets"));
```

Comment:
I tried updating this code in a few different ways but ultimately, this felt like the least-invasive way to make the change. I don't fully understand what this code is doing (and didn't really bother to figure out), so, someone else can change this code later if they find a better way of doing it.

Plus, a panic is an ICE anyway, so, if this causes any noticeable difference in behaviour it was a bug anyway.